CHANGELOG.md

```diff
@@ -1,4 +1,9 @@
-# dbt_iterable v0.RELEASE.RELEASE
+# dbt_iterable v0.11.1
+[PR #44](https://github.com/fivetran/dbt_iterable/pull/44) includes the following updates:
+
```

Comment:
I'd add more about the `experiment_id` updates (plus the new columns you're adding) in either a Breaking Change section, or in Bug Fixes below, whichever makes more sense.  It looks like it's mostly related to the `iterable__using_event_extension` section, but more context would be helpful since the logic introduced is non-intuitive.

Reply:
Thanks Avinash, I added more info in Under the Hood for `iterable__using_event_extension` and the related `experiment_id` field